client/signup/steps/emails/index.jsx

```diff
@@ -118,9 +119,16 @@ class EmailsStep extends Component {
 	}
 
 	render() {
-		const { flowName, translate, stepName, positionInFlow, signupDependencies, hideBackButton } =
-			this.props;
-		const backUrl = 'start/domains/';
+		const {
+			backUrl = 'domains/',
```

Comment:
@fredrikekelund Thanks for fixing this!

Reply:
Initially I thought this fixes the issue where the "back" button at the email step opens a blank page. Looks like the fix has to be done in the steps config where we pass `backUrl`: https://github.com/Automattic/wp-calypso/pull/67046/files#r956918728